ntcore/src/generate/java/NetworkTableInstance.java.jinja

```diff
@@ -951,6 +952,19 @@ public final class NetworkTableInstance implements AutoCloseable {
     return NetworkTablesJNI.isConnected(m_handle);
   }
 
+  /**
+   * Get the time offset between server time and local time. Add this value to local time to get
+   * the estimated equivalent server time. In server mode, this always returns 0. In client mode,
```

Comment:
The Java doc says "server mode", whereas the C++ doc says "non-client modes". Should they match?

Reply:
Whoops I neglected to update the NetworkTableInstance C++ doc (I got the others).  Thanks.